deploy/uninstall.yml

```diff
@@ -60,48 +60,48 @@
         state: absent
       ignore_errors: yes
 
-- hosts: executionnode1
+- hosts: en1
   become: yes
   tasks:
     - name: Stop and remove execution_node container 1
       docker_container:
-        name: "execution-node-{{ inventory_hostname }}-{{ port_1 }}"
+        name: "execution-node-{{ ansible_host }}-{{ port_1 }}"
```

Comment:
Here port_1 should be replaced with port. Same applies for all ports. One question, can we run specify `hosts: execution nodes` and let it delete all the execution nodes from the inventory file. The last step of deleting the /opt/autolabjs and removing the execution_node image directory can be moved down.